dist/chota.css

```diff
@@ -629,6 +629,21 @@ button:disabled:hover {
 ::placeholder {
   color: #bdbfc4;
 }
+.group {
+	display: -webkit-box;
+	display: -ms-flexbox;
+	display: flex; /* hide space between element */
+}
+.group>* {
```

Comment:
I would refrain from using a `*` and specify the selectors of the components like input, button and select here.

Reply:
I changed it to:
```css

.grouped > select,
.grouped > input,
.grouped > button,
.grouped > .button {
  margin: 0 0 0 -1px !important;
  border-radius:0 !important;
}
.grouped > select:first-child,
.grouped > input:first-child,
.grouped > button:first-child,
.grouped > .button:first-child {
  margin: 0 !important;
  border-radius:4px 0 0 4px !important;
}

.grouped > select:last-child,
.grouped > input:last-child,
.grouped > button:last-child,
.grouped > .button:last-child {
  border-radius:0 4px 4px 0 !important;
}
```
But I don't see why we can't let the users decide what they want inside they groups.

Reply:
Actually you are right, this will just make the CSS heavier, we can just use what you initially did